pkg/lifecycle/terraform/post_terraform.go

```diff
@@ -0,0 +1,149 @@
+package terraform
+
+import (
+	"fmt"
+	"path"
+
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/ship/pkg/api"
+	"github.com/replicatedhq/ship/pkg/templates"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"gopkg.in/yaml.v2"
+)
+
+// A mostly minimal representation of a kube config as per -
+// https://github.com/kubernetes/client-go/blob/19c591bac28a94ca793a2f18a0cf0f2e800fad04/tools/clientcmd/api/types.go
```

Comment:
the structure follows the types but is structured differently in order to write the `kubeconfig` correctly